revpi-hat-eep/src/gpio.rs

```diff
@@ -6,7 +6,9 @@ use rpi_hat_eep::gpio_map;
 use serde::{Deserialize, Serialize};
 use std::fmt::Display;
 
-const MAX_GPIOS: usize = 28;
+const BANK0_GPIOS: usize = 28;
+const BANK1_GPIOS: usize = 18;
+const MAX_GPIOS: usize = BANK0_GPIOS + BANK1_GPIOS;
```

Comment:
Duplicated constants, are alreadey defined in rpi-hat-eep/src/gpio.rs
Hard to maintain. Try to do something like this -> use rpi-hat-eep::BANK0_GPIOS or whatever is appropriate to remove duplication.

Some use constant-only files.